README.md

```diff
@@ -1,2 +1,58 @@
-# hellomicroverse
-set up hello microverse project
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b> README </b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [hello microverse] 
+
+> Describe your project in 1 or 2 sentences.
+
+**[hello microverse]** 
+
+## 🛠 Built With css and html
+
```

Comment:
- [ ] Good job Using the Microverse readme file template suggested by the last reviewer. 👍 👍 .
But according to the project requirement, you are meant to customize the readme file to your project,
> Add descriptive README file to your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.
- Please consider editing the Readme file to look professional. 😄 🛩️ This is an example of how your readme file should look like after customizing it [Proffessional Readme](https://github.com/V-Blaze/Mobile_menu_project/blob/Mobile_menu/README.md)